### PR TITLE
zlib upgrade for CVE-2018-25032

### PR DESCRIPTION
**Pull request by oxelson**
Upgrade to 1.2.12

### CODE REVIEW

**Comment by akrherz** [comment]
Thanks @oxelson , but doesn't the actual zlib 1.2.12 source need included as well?

**Comment by oxelson** [comment]
*le sigh* Sorry @akrherz! It's been a day.  I'll get that added. 🤪

**Comment by akrherz** [comment]
Whelp, CI is green here, but is 1) not good and 2) makes me realize what the docker images were doing with extlibs.  Ufff

**Comment by oxelson** [comment]
I double checked and did grab the source code for the latest zlib and put in the symlink -- but the v1.2.12 directory & contents were not pushed to the repo...?  Sorry about that.  I'll correct it.

> Whelp, CI is green here, but is 1) not good and 2) makes me realize what the docker images were doing with extlibs. Ufff

How did you want to proceed?

**Comment by akrherz** [comment]
@oxelson I am unsure.  It would certainly be nice just to jettison zlib from extlibs and let the OS provide the library, but that may be a bridge too far.

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Building generates lots of git diff, so ignored
-extlibs/
+#extlibs/
```
Comment (akrherz) [inline .gitignore:2]: Un-necessary, just use `git -f add`

**Comment by akrherz** [comment]
> we should probably rebuild gempak

We are not distributing binaries at this time, I am still boggling this.

**Comment by oxelson** [comment]
> We are not distributing binaries at this time, I am still boggling this.

Sounds good. 👍

**Comment by akrherz** [comment]
Thanks again @oxelson , will take this once CI passes.

**Pull request merged**